src/main/java/com/vmware/vipclient/i18n/messages/api/opt/server/ComponentsBasedOpt.java

```diff
@@ -63,9 +63,7 @@ public JSONObject queryFromServer(final Set<String> components, final Set<String
 
         int statusCode = this.getResponseCode(this.responseJsonObj);
         if (!this.isSuccess(statusCode))
-            throw new VIPJavaClientException(
-                    String.format(ConstantsMsg.SERVER_RETURN_ERROR, statusCode, this.getResponseMessage(this.responseJsonObj)));
-
-        return this.responseJsonObj;
+            logger.warn(String.format(ConstantsMsg.SERVER_RETURN_ERROR, statusCode, this.getResponseMessage(this.responseJsonObj)));
```

Comment:
The message is name is ConstantsMsg.SERVER_RETURN_ERROR, the log level is warn. please confirm it.